controllers/common.go

```diff
@@ -1441,16 +1441,27 @@ func (r *MultiClusterHubReconciler) addPluginToConsole(multiClusterHub *operator
 		console.Spec.Plugins = []string{}
 	}
 
+	modifiedSpec := false
 	// Add acm to the plugins list if it is not already there
 	if !utils.Contains(console.Spec.Plugins, "acm") {
-		log.Info("Ready to add plugin")
+		log.Info("Ready to add acm plugin")
 		console.Spec.Plugins = append(console.Spec.Plugins, "acm")
+		modifiedSpec = true
+	}
+	// Add mce to the plugins list if it is not already there
+	if !utils.Contains(console.Spec.Plugins, "mce") {
+		log.Info("Ready to add mce plugin")
+		console.Spec.Plugins = append(console.Spec.Plugins, "mce")
+		modifiedSpec = true
+	}
+
+	if modifiedSpec {
 		err = r.Client.Update(ctx, console)
 		if err != nil {
-			log.Info("Failed to add acm consoleplugin to console")
+			log.Info("Failed to add consoleplugin to console")
 			return ctrl.Result{Requeue: true}, err
 		} else {
-			log.Info("Added acm consoleplugin to console")
+			log.Info("Added consoleplugin to console")
```

Comment:
Do not indent normal code when handling errors:
https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#indent-error-flow